packages/pure/src/mc-usb-device/usb-parser.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { UsbResponse, WriteOption } from "./usb-device.facade.class"
+import { LoggerFactory } from "../logger"
+
+const logger = LoggerFactory.getInstance()
+
+export class UsbParser {
+  static buildContainerPacket(option: WriteOption): ArrayBuffer {
+    const packetLength = 12 + option.payload.length * 4
+
+    const buffer = new ArrayBuffer(packetLength)
+    const bytes = new DataView(buffer)
+    bytes.setUint32(0, packetLength, true)
+    bytes.setUint16(4, option.type, true)
+    bytes.setUint16(6, option.code, true)
+    bytes.setUint32(8, option.id, true)
+
+    option.payload.forEach((element, index) => {
+      bytes.setUint32(12 + index * 4, element, true)
+    })
+    logger.info(`==== usb: buildContainerPacket: ====`)
+    console.log(buffer)
+    return buffer
+  }
+
+  static parseContainerPacket(buffer: ArrayBuffer): UsbResponse {
+    logger.info(`==== usb: parseContainerPacket: ====`)
+    console.log(buffer)
```

Comment:
Please remove it

Reply:
https://github.com/mudita/mudita-center/pull/697/commits/32a9172dd4cf1699b3d0d7babaa727fdce4e8e10